.prettierignore

```diff
@@ -0,0 +1,6 @@
+.hars
+.vscode
+.svelte-kit/
+.svelte-kit/tsconfig.json
```

Comment:
I wasn't there when you discussed this, almost all of this seems sensible to ignore given it's auto generated / compiled code. However not sure about this line - I'm guessing there's a reason for it, just curious as to why we want to exclude this as this seems like a file we maintain?

Reply:
I'm curious about this too. Is there a reason why we have to be extra specific about this file? Just ignoring the directory wasn't giving us what we want?

Reply:
its not working and always gets formatted, thats mostly why. trying to figure this out lol

Reply:
ive fixed this now.